src/Core/src/Platform/Windows/MauiStepper.cs

```diff
@@ -0,0 +1,267 @@
+﻿#nullable disable
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Media;
+using Microsoft.Maui.Graphics;
+using WBrush = Microsoft.UI.Xaml.Media.Brush;
+using WVisualStateManager = Microsoft.UI.Xaml.VisualStateManager;
+using WVisualStateGroup = Microsoft.UI.Xaml.VisualStateGroup;
+using WVisualState = Microsoft.UI.Xaml.VisualState;
+
+namespace Microsoft.Maui
+{
+	public sealed class MauiStepper : Control
+	{
+		public static readonly DependencyProperty ValueProperty = 
+			DependencyProperty.Register("Value", typeof(double), typeof(MauiStepper), new PropertyMetadata(default(double), OnValueChanged));
+
+		public static readonly DependencyProperty MaximumProperty = 
+			DependencyProperty.Register("Maximum", typeof(double), typeof(MauiStepper), new PropertyMetadata(default(double), OnMaxMinChanged));
+
+		public static readonly DependencyProperty MinimumProperty = 
+			DependencyProperty.Register("Minimum", typeof(double), typeof(MauiStepper), new PropertyMetadata(default(double), OnMaxMinChanged));
+
+		public static readonly DependencyProperty IncrementProperty = 
```

Comment:
@hartez was this ever renamed? I remember some discussion somewhere about potentially renaming to Interval instead

Reply:
I believe we renamed it in MAUI Core. I'm less concerned with what it's called here on the native platform, but if it's not a breaking change then it might be nice to keep it consistent.